protocol-units/bridge/setup/src/local.rs

```diff
@@ -4,31 +4,12 @@ use aptos_sdk::types::LocalAccount;
 use bridge_config::common::eth::EthConfig;
 use bridge_config::common::movement::MovementConfig;
 use bridge_config::common::testing::TestingConfig;
-use bridge_config::Config as BridgeConfig;
-use hex::ToHex;
 use rand::prelude::*;
-use std::{
-	env, fs,
-	io::Write,
-	path::PathBuf,
-	process::{Command, Stdio},
-};
+use std::process::Stdio;
 use tokio::io::AsyncBufReadExt;
 use tokio::io::BufReader;
 use tokio::process::Command as TokioCommand;
 
-pub async fn setup(
-	mut config: BridgeConfig,
-) -> Result<(BridgeConfig, AnvilInstance), anyhow::Error> {
-	tracing::info!("Bridge local setup");
-	//Eth init: Start anvil.
-	let anvil = setup_eth(&mut config.eth, &mut config.testing);
-	tracing::info!("Bridge after anvil");
-
-	//By default the setup deosn't start the Movement node.
-	Ok((config, anvil))
-}
-
 pub fn setup_eth(config: &mut EthConfig, testing_config: &mut TestingConfig) -> AnvilInstance {
```

Comment:
Do we still want this `local.rs` file, if `setup_eth` is the only `fn` in it?

Could `setup_eth` be moved somewhere else, perhaps into `deploy.rs`, and we remove `local.rs`?

Reply:
Ah I wasn't looking at the whole file, I see what you mean now.